### PR TITLE
Add mysql and postgres to the Travis-CI test suite

### PR DESCRIPTION
**Pull request by cmdcolin**
This is an initial pull request to add mysql and postgres tests to be run on travis CI.

Reference https://github.com/GMOD/Apollo/issues/789

In progress :)

### CODE REVIEW

**Comment by cmdcolin** [comment]
You can see here that instead of one build, it launches 3 now https://travis-ci.org/GMOD/Apollo/builds/103704982

File: config/mysql.travis [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+environments {
+    test {
+        dataSource{
+            dbCreate = "update" // one of 'create', 'create-drop', 'update', 'validate', ''
```
Comment (nathandunn) [inline config/mysql.travis]: this should always be create-drop

**Comment by nathandunn** [comment]
Overall looks great.  I'm not sure why the postgresql environment is failing.  'apollo-test' is by default what we usually use (but not in this environment, which is typically H2), but I don't see it in your code, so not sure what is up.  Dump the cache and start over?

File: config/mysql.travis (hunk 1 above)
Comment (cmdcolin) [inline config/mysql.travis]: yep. fixed

**Comment by cmdcolin** [comment]
I just made it create database name apollo in .travis.yml so postgres should pass tests now

**Comment by nathandunn** [comment]
This is great!  

Any idea why we can't see the log files for this?   Maybe I'm missing a link.

https://travis-ci.org/GMOD/Apollo/builds/103724257

Also, do the concurrent builds share the log?     Clicking on it directly seems to go nowhere. 

This is what I'm seeing:

screen shot 2016-01-20 at 3 23 03 pm

**Comment by cmdcolin** [comment]
travis-ci has had some bumps today so log files can't be viewed (tests are still being run though) https://www.traviscistatus.com/

hopefully they recover soon :)

**Comment by nathandunn** [comment]
It looks like it loaded.  I think this will be very helpful.  :+1:

**Pull request merged**